package.json

```diff
@@ -52,7 +52,7 @@
     "webpack-config-single-spa-ts": "^4.1.2",
     "webpack-dev-server": "^4.9.3",
     "webpack-merge": "^5.8.0",
-    "webpack-pwa-manifest": "^4.3.0"
+    "webpack-pwa-manifest": "^4.0.0"
```

Comment:
Was this an intentional downgrade? Was it causing some issue?

Reply:
@nmorasb This one was due to the npm audit fix, should I revert it?

Reply:
@nmorasb I reverted the package files, thanks!